# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * account_invoice_constraint_chronology
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: account-financial-tools (8.0)\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-01-30 16:31+0000\n"
"PO-Revision-Date: 2018-10-11 11:16+0000\n"
"Last-Translator: Rudolf Schnapka <rs@techno-flex.de>\n"
"Language-Team: German (http://www.transifex.com/oca/OCA-account-financial-"
"tools-8-0/language/de/)\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: Weblate 3.1.1\n"

#. module: account_invoice_constraint_chronology
#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_journal__check_chronology
msgid "Check Chronology"
msgstr "Prüfe Chronologie"

#. module: account_invoice_constraint_chronology
#: code:addons/.src/account-financial-tools/account_invoice_constraint_chronology/model/account_move.py:0
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, fuzzy, python-format
msgid ""
"Chronology conflict: A conflicting draft invoice dated before {date_invoice} "
"exists, please validate it first."
msgstr ""
"Chronolgiefehler. Geben Sie Rechnungsentwürfe vor %s frei und versuchen es "
"erneut."

#. module: account_invoice_constraint_chronology
#: code:addons/.src/account-financial-tools/account_invoice_constraint_chronology/model/account_move.py:0
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, fuzzy, python-format
msgid ""
"Chronology conflict: A conflicting validated invoice dated after "
"{date_invoice} exists."
msgstr "Chronologiefehler. Es gibt mind. eine Rechnung mit Datum nach %s."

#. module: account_invoice_constraint_chronology
#: code:addons/.src/account-financial-tools/account_invoice_constraint_chronology/model/account_move.py:0
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, fuzzy, python-format
msgid ""
"Chronology conflict: An invoice with a higher number {highest_name} dated "
"before {date_invoice} exists."
msgstr "Chronologiefehler. Es gibt mind. eine Rechnung mit Datum nach %s."

#. module: account_invoice_constraint_chronology
#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_journal__display_name
#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_move__display_name
msgid "Display Name"
msgstr ""

#. module: account_invoice_constraint_chronology
#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_journal__id
#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_move__id
msgid "ID"
msgstr ""

#. module: account_invoice_constraint_chronology
#: model:ir.model,name:account_invoice_constraint_chronology.model_account_journal
msgid "Journal"
msgstr "Journal"

#. module: account_invoice_constraint_chronology
#: model:ir.model,name:account_invoice_constraint_chronology.model_account_move
#, fuzzy
msgid "Journal Entry"
msgstr "Journal"

#. module: account_invoice_constraint_chronology
#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_journal____last_update
#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_move____last_update
msgid "Last Modified on"
msgstr ""

#~ msgid "Invoice"
#~ msgstr "Rechnung"
